benches/glif_parse.rs

```diff
@@ -63,8 +67,9 @@ pub fn criterion_benchmark(c: &mut Criterion) {
     // many glyphs
     c.bench_function("parse MutatorSansLightWide glyphs", |b| {
         let glyphs = load_all(MUTATOR_SANS_GLYPHS_DIR);
+        assert_eq!(glyphs.len(), 48);
```

Comment:
This makes for fragile tests; I suggest a check that won't snap if we alter mutator sans. len >= 25 perhaps.